mw-webapp/src/utils/LocalStorageWorker.ts

```diff
@@ -190,3 +195,12 @@ export type AllUsersPageSettings = {
    */
   view: View;
 }
+
+export type SettingPageSettings = {
+
+  /**
+   * If true notifications is allowed
+   * @default false
+   */
+  isNotificationAllowed: boolean;
```

Comment:
use OSNotification to separate onsite and os notifications

Reply:
renaming

Reply:
Done